python/tchannel/messages/common.py

```diff
@@ -32,3 +38,9 @@ class ChecksumType(IntEnum):
         ChecksumType.farm32: rw.number(4),  # csum:4
     }
 )
+
+
+checksum_msg_types = [Types.CALL_REQ,
```

Comment:
Use CAPS for top-level constants like this.

Also, make it a `frozenset` instead of a list.

    frozenset([...])